### PR TITLE
[PLAYER-4937] Error screen for audio-only

### PR DESCRIPTION
**Pull request by isheftel-ooyala**
Design was implemented according to this ticket

https://jira.corp.ooyala.com/browse/PLAYER-3457

### CODE REVIEW

File: sdk/react/ooyalaSkinPanelRenderer.js [hunk 1]
```diff
@@ -90,7 +91,9 @@ OoyalaSkinPanelRenderer.prototype.renderErrorScreen = function() {
     <ErrorScreen
       error={this.skin.state.error}
       localizableStrings={this.skin.props.localization}
-      locale={this.skin.props.locale}>
+      locale={this.skin.props.locale}
+      isAudioOnly={this.skin.state.screenType === SCREEN_TYPES.ERROR_SCREEN_AUDIO}
```
Comment (maxkupetskii) [inline sdk/react/ooyalaSkinPanelRenderer.js:95]: We have a `contentType` in `OoyalaSkin` for this purpose I think.

File: sdk/react/panels/ErrorScreen.js [hunk 2]
```diff
@@ -36,6 +41,16 @@ class ErrorScreen extends React.Component {
       </Text>);
   };
 
+  getTitleAudioOnly = () => {
+    var title = "unplayable content error";
```
Comment (maxkupetskii) [inline sdk/react/panels/ErrorScreen.js]: Why var?

File: sdk/react/panels/MoreDetailsScreen.js [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+
+import React from 'react';
+import {Animated, ScrollView, Text, View,} from 'react-native';
+
+import {BUTTON_NAMES, ERROR_MESSAGE, SAS_ERROR_CODES,} from '../constants';
+
+const Utils = require('../utils');
+const styles = Utils.getStyles(require('./style/moreDetailsScreenStyles.json'));
+
+var Log = require('../log');
```
Comment (maxkupetskii) [inline sdk/react/panels/MoreDetailsScreen.js]: We may `import Log from (...)`

File: sdk/react/panels/MoreDetailsScreen.js [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+      Utils.localizedString(this.props.locale, title, this.props.localizableStrings).toUpperCase();
+    return (
+      <Text style={styles.title}>
+        {localizedTitle}
+      </Text>);
+  };
+
+  _renderErrorDescription = () => {
+    if (this.props.error && this.props.error.description) {
+      var userInfo = this.props.error.userInfo || {};
```
Comment (maxkupetskii) [inline sdk/react/panels/MoreDetailsScreen.js]: Same about vars

**Review by maxkupetskii — CHANGES_REQUESTED** [review]

File: sdk/react/ooyalaSkinPanelRenderer.js (hunk 1 above)
Comment (isheftel-ooyala) [reply sdk/react/ooyalaSkinPanelRenderer.js:95]: Discussed. Bridge for iOS edited

File: sdk/react/panels/ErrorScreen.js (hunk 2 above)
Comment (isheftel-ooyala) [reply sdk/react/panels/ErrorScreen.js]: Fixed

File: sdk/react/panels/MoreDetailsScreen.js (hunk 3 above)
Comment (isheftel-ooyala) [reply sdk/react/panels/MoreDetailsScreen.js]: Fixed

File: sdk/react/panels/MoreDetailsScreen.js (hunk 4 above)
Comment (isheftel-ooyala) [reply sdk/react/panels/MoreDetailsScreen.js]: Fixed

File: sdk/iOS/OoyalaSkinSDK/OOSkinPlayerObserver.m [hunk 5]
```diff
@@ -328,7 +328,9 @@ - (void)bridgeErrorNotification:(NSNotification *)notification {
 
   NSDictionary *eventBody = @{codeKey:        code,
                               descriptionKey: detail,
-                              userInfoKey:    userInfo};
+                              userInfoKey:    userInfo,
+                              screenTypeKey:  OOStreamPlayer.defaultPlayerInfo.isAudioOnly ?
+                                              @“audio” : @“video”};
```
Comment (maxkupetskii) [inline sdk/iOS/OoyalaSkinSDK/OOSkinPlayerObserver.m]: I bet these are wrong quotes, it should be
`" "`

Comment (isheftel-ooyala) [reply sdk/iOS/OoyalaSkinSDK/OOSkinPlayerObserver.m]: Fixed

**Review by maxkupetskii — APPROVED** [review]

**Review by SergeyBicarte — APPROVED** [review]
LGTM

**Pull request merged**